iree/compiler/Codegen/Passes.td

```diff
@@ -159,10 +159,18 @@ def LLVMCPUUnfuseFMAOps :
   let constructor = "mlir::iree_compiler::createLLVMCPUUnfuseFMAOpsPass()";
 }
 
-def VectorToAArch64InlineAsm :
-    Pass<"iree-llvmcpu-vector-to-aarch64-inline-asm", "FuncOp"> {
-  let summary = "Convert vector operations to aarch64 inline asm LLVMIR dialect";
-  let constructor = "mlir::iree_compiler::createVectorToAArch64InlineAssemblyPass()";
+def VectorContractCustomKernels :
+    Pass<"iree-llvmcpu-vector-contract-custom-kernels", "FuncOp"> {
+  let summary = "Enable custom kernels (inline assembly or intrinsics) for some vector.contract ops";
+  let constructor = "mlir::iree_compiler::createVectorContractCustomKernelsPass()";
```

Comment:
Hmm I think having declared this here you should be able to include a genned thingy in Passes.h, but I see nowhere else is doing that yet, so don't worry about it for now

Reply:
ah ok (so not doing anything for now).